src/java/fr/paris/lutece/plugins/forms/export/pdf/FormResponsePdfExport.java

```diff
@@ -120,7 +120,7 @@ private List<PdfCell> createCellsForStep( FormResponseStep formResponseStep )
     {
         Step step = formResponseStep.getStep( );
 
-        List<FormDisplay> listStepFormDisplay = FormDisplayHome.getFormDisplayListByParent( step.getId( ), 0 );
+        List<FormDisplay> listStepFormDisplay = FormDisplayHome.getFormDisplayListOrderByQuestionExportDisplayOrder( step.getId( ) );
```

Comment:
The new method does not make it possible to properly take groups into account. Indeed, as we retrieve all the FormDisplay with getFormDisplayListOrderByQuestionExportDisplayOrder, the questions contained in groups are displayed several times because they are displayed both directly in createCellsForStep and each time the createCellsForGroup method is called for the groups to which they belong .
It would therefore be necessary to recover the FormDisplay according to their parent like with getFormDisplayListByParent but also to see how to put an order on the groups because for the moment we only do it on the questions.

Reply:
Correction on new PR : https://github.com/lutece-platform/lutece-form-plugin-forms/pull/228